src/app/app.component.html

```diff
@@ -1,134 +1,10 @@
-                Women & Girls Match Fund
-              </a>
-            </li>
-            <li>
-              <a href="https://blog.thebiggive.org.uk/mental-health-match-fund/" class="icon-mental-health">
-                Mental Health Match Fund
-              </a>
-            </li>
-            <li>
-              <a href="https://blog.thebiggive.org.uk/anchor-match-fund/" class="icon-anchor-match">
-                Anchor Match Fund
-              </a>
-            </li>
-            <li>
-              <a href="https://blog.thebiggive.org.uk/emergency-campaigns/" class="icon-emergency">
-                Emergency Match Fund
-              </a>
-            </li>
-          </ul>
-        </li>
-        <li>
-          <a href="https://blog.thebiggive.org.uk/run-your-own-campaign/">Run your match funding campaign</a>
-        </li>
-      </ul>
-    </li>
-    <li>
-      <a (click)="noNav($event)">
-        About Us
-        <biggive-misc-icon class="bx bxs-chevron-down sub-menu-arrow arrow" background-colour="white" icon-colour="black" icon="CaretRight"></biggive-misc-icon>
-      </a>
-      <ul class="sub-menu">
-        <li>
-          <a href="https://blog.thebiggive.org.uk/our-people">Our People</a>
-        </li>
-        <li>
-          <a href="https://blog.thebiggive.org.uk/our-story">Our Story</a>
-        </li>
-        <li>
-          <a href="https://blog.thebiggive.org.uk/our-community">Our Community</a>
-        </li>
-        <li>
-          <a href="https://blog.thebiggive.org.uk/our-fees">Our Fees</a>
-        </li>
-        <li>
-          <a href="https://blog.thebiggive.org.uk/faqs">FAQs</a>
-        </li>
-      </ul>
-    </li>
-    <li>
-      <a (click)="noNav($event)">
-        Resources
-        <biggive-misc-icon class="bx bxs-chevron-down sub-menu-arrow arrow" background-colour="white" icon-colour="black" icon="CaretRight"></biggive-misc-icon>
-      </a>
-      <ul class="sub-menu">
-        <li><a href="https://blog.thebiggive.org.uk/case-studies">Case Studies</a></li>
-        <li><a href="https://blog.thebiggive.org.uk/blog">Blog</a></li>
-        <li><a href="https://blog.thebiggive.org.uk/reports-insights">Reports &amp; Insights</a></li>
-        <li><a href="https://blog.thebiggive.org.uk/press">Press</a></li>
-      </ul>
-    </li>
-  </ul>
-  <ul slot="nav-secondary">
-    <li *ngIf="isLoggedIn && flags.profilePageEnabled">
-      <a routerLink="/my-account">My Account</a>
-    </li>
-    <li><a href="https://www.thebiggive.org.uk/s/contact-us">Contact us</a></li>
-    <li><a href="https://www.thebiggive.org.uk/charities/s/login">Charity login</a></li>
-  </ul>
-</biggive-main-menu>
+<biggive-main-menu
+  use-preset-menu-content="true"
+  donate-url-prefix="https://donate.thebiggive.org.uk"
+  blog-url-prefix="https://blog.thebiggive.org.uk"
+  experience-url-prefix="https://www.thebiggive.org.uk"
```

Comment:
these three were already hardcoded but I think I may was well take the chance to make them environment appropriate while before I merge this.